app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -0,0 +1,174 @@
+                      dense
+                      flat
+                      outlined
+                      solo
+                    ></v-text-field>
+                  </template>
+                  <v-date-picker
+                    v-model="startDate"
+                    data-test="picker-form-startDate"
+                    @input="startDateMenu = false"
+                    :readonly="reviewMode"
+                  ></v-date-picker>
+                </v-menu>
+              </v-col>
+
+              <v-col cols="12" sm="6" offset-sm="0" offset-md="1" md="4">
+                <v-menu
+                  v-model="endDateMenu"
+                  data-test="menu-form-endDate"
+                  :close-on-content-click="true"
+                  :nudge-right="40"
+                  transition="scale-transition"
+                  offset-y
+                  min-width="290px"
+                >
+                  <template v-slot:activator="{ on }">
+                    <label>To</label>
+                    <v-text-field
+                      v-model="endDate"
+                      placeholder="yyyy-mm-dd"
+                      append-icon="event"
+                      v-on:click:append="endDateMenu = true"
+                      readonly
+                      v-on="on"
+                      dense
+                      flat
+                      outlined
+                      solo
+                    ></v-text-field>
+                  </template>
+                  <v-date-picker
+                    v-model="endDate"
+                    data-test="picker-form-endDate"
+                    @input="endDateMenu = false"
+                    :readonly="reviewMode"
+                  ></v-date-picker>
+                </v-menu>
+              </v-col>
+            </v-row>
+          </div>
+          <p :class="!dateRange ? 'mt-5' : ''">
+            File Name and Type: <strong>{{ fileName }}</strong>
+          </p>
+          <p>
+            <small class="text--disabled">
+              * The export data feature works well for simple form designs that
+              don't contain complex nested arrays of form components. If you
+              make changes to your form design the structure of your export will
+              also change. We therefore caution against implementing automation
+              with other systems without accounting for these factors.
+            </small>
+          </p>
+        </v-card-text>
+
+        <v-card-actions class="justify-center">
+          <v-btn class="mb-5 mr-5" color="primary" @click="callExport">
+            <span>Export</span>
+          </v-btn>
+          <v-btn class="mb-5" outlined @click="dialog = false">
+            <span>Cancel</span>
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </span>
+</template>
+
+<script>
+import moment from 'moment';
+import { mapGetters } from 'vuex';
+import formService from '@/services/formService.js';
```

Comment:
Going to start advocating importing our services via index instead of directly referencing the file
```json
import { formService } from '@/services';
```

Reply:
We can go round and update them all